runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -4853,7 +4853,7 @@ class INTERPRETER_CLASS
 		return rc;
 	}
 
-	/* java.lang.Object: public final native void wait(long millis, int nanos); */
+	/* java.lang.Object: public final native void waitImpl(long millis, int nanos); */
 	VMINLINE VM_BytecodeAction
 	inlObjectWait(REGISTER_ARGS_LIST)
```

Comment:
Should really change the method name to match the native name (no functional effect, just consistency).

Reply:
updated INL name to `inlObjectWaitImpl`